From 760a8628890b38a1f6c8c13036bdfce6d47a4d29 Mon Sep 17 00:00:00 2001
From: Eyal Harari <Eyal.Harari@freesacle.com>
Date: Tue, 29 Jul 2014 09:15:47 +0300
Subject: [PATCH 757/987] DSAR: configure tcp flags in DSAR tables

Signed-off-by: Eyal Harari <Eyal.Harari@freesacle.com>
[Lu:Original patch taken from
QorIQ-SDK-V1.7-SOURCE-20141218-yocto.iso]
Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 drivers/net/ethernet/freescale/fman/Peripherals/FM/Port/fm_port.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/freescale/fman/Peripherals/FM/Port/fm_port.c b/drivers/net/ethernet/freescale/fman/Peripherals/FM/Port/fm_port.c
index f28d6b4..920749b 100644
--- a/drivers/net/ethernet/freescale/fman/Peripherals/FM/Port/fm_port.c
+++ b/drivers/net/ethernet/freescale/fman/Peripherals/FM/Port/fm_port.c
@@ -5736,6 +5736,7 @@ t_Error FM_PORT_EnterDsar(t_Handle h_FmPortRx, t_FmPortDsarParams *params)
         if (params->p_AutoResFilteringInfo->tcpPortDropOnHit)
             tmp |= TCP_PORT_TBL_PASS_MASK;
         WRITE_UINT8(ArCommonDescPtr->filterControl, tmp);
+        WRITE_UINT16(ArCommonDescPtr->tcpControlPass, params->p_AutoResFilteringInfo->tcpFlagsMask);
 
         // ip filtering
         if (params->p_AutoResFilteringInfo->ipProtTableSize)
-- 
1.9.1

